src/ui-test/camelProjectOfferingTest.ts

```diff
@@ -0,0 +1,78 @@
+import { QuickOpenBox, InputBox, QuickPickItem } from "vscode-extension-tester";
+
+import { openCommandPrompt, typeCommandConfirm, convertArrayObjectsToText, convertArrayObjectsToTextAndDescription } from "./common/commonUtils";
+
+import { assertEqualOptions } from "./common/testUtils";
+
+import { ProjectInitializer } from "./common/projectInitializerConstants";
+import { Catalog } from "../Catalog";
+import { expect } from "chai";
+
+const CAMEL_MISSIONS_EXPECTED = [
+    'circuit-breaker', 'configmap', 'health-check', 'rest-http', 'istio-distributed-tracing'
+];
+
+/**
+ * @author odockal@redhat.com
+ */
+export function testFuseProjectOffering() {
+    describe('Verify Project initializer Camel/Fuse Command palette options', async function() {
+
+        let inputBox: QuickOpenBox;
+        let catalogFuse: any;
+
+        before(async function() {
+            this.timeout(5000);
+            let catalogBuilder = new Catalog('https://forge.api.openshift.io/api/');
```

Comment:
would be better to use the value that is available in the runtime code?
In case, the url is modified, the test will be no more testing the right endpoint, isn't it?

Reply:
You are right and I agree, BUT. If it would be available from extension itself then yes, pretty easy. Since it is hardcoded in extension I doubt that... Only way I can get it is from settings.json on the fs (which differs on every os), or using UI (implement it using extester). Question is if it is worth it. Is it gonna change anytime soon?

Or is there other way to get it from runtime?

Reply:
maybe we can expose the catalogBuilder from the extension?
https://github.com/redhat-developer/vscode-project-initializer/blob/13a2f372ed9c1e0affcf11dc586a998e2d7936f5/src/extension.ts#L10

or maybe worth creating a constant that is exposed for `"https://forge.api.openshift.io/api/"`as it is already duplicated in 4 places

it should be also possible to retrieve the default value of "project.initializer.endpointUrl", https://github.com/redhat-developer/vscode-project-initializer/blob/13a2f372ed9c1e0affcf11dc586a998e2d7936f5/package.json#L72
If preference not modified elsewhere, It should be possible with something like `vscode.workspace.getConfiguration("project.initializer").inspect<string>("endpointUrl")`

but as it seems to require changes in several places to have it clean, it might be better to do it in a separated PR

Reply:
Yes, probably exposing it "not sure about implementation" should do the work. 

Though vscode.workspace.etc.. does not work as tests do not have access to vscode runtime at the moment, I tried this.

Reply:
As of now, I have created constant for catalog builder url in tests. If we change extension to expose this variable I would be happy to change it. If it is ok for now? I am still thinking of another batch of tests to be done, it could be included.

Reply:
ok